File: fizzbuzz.cs
// Language:       C#
// Web site:       http://www.ecma-international.org/publications/standards/Ecma-334.htm
// Last tested on: Ubuntu 22.04.4
// Requires:       apt-get mono-xbuild mono-mcs mono-devel
//                 (was apt-get install mono-gmcs)

using System;
class FizzBuzz {
    static void Main() {
        for (int n = 1; n <= 100; n ++) {
            if (n % 15 == 0) {
                Console.WriteLine("FizzBuzz");
            }
            else if (n % 3 == 0) {
                Console.WriteLine("Fizz");
            }
            else if (n % 5 == 0) {
                Console.WriteLine("Buzz");
            }
            else {
                Console.WriteLine(n);
            }
        }
    }
}

File: fizzbuzz.vb
' Language:       Visual Basic .NET
' Web site:       https://learn.microsoft.com/en-us/dotnet/visual-basic/programming-guide/
' Last tested on: Ubuntu 22.04.3
' Requires:       apt-get install mono-vbnc

Module FizzBuzz
    Sub Main()
        Dim I As Integer
        For I = 1 to 100
            If I mod 15 = 0 Then
                Console.WriteLine("FizzBuzz")
            ElseIf I mod 3 = 0 Then
                Console.WriteLine("Fizz")
            ElseIf I mod 5 = 0 Then
                Console.WriteLine("Buzz")
            Else
                Console.WriteLine(I)
            End If
        Next
    End Sub
End Module
